# msg/JointPlotData.msg
float64 measured_q1
float64 desired_q1
float64 measured_q2
float64 desired_q2
